config/routes.rb

```diff
@@ -1,3 +1,7 @@
 Rails.application.routes.draw do
+  resources :members
+  resources :groups
+  resources :contacts
+  devise_for :users
```

Comment:
It could be convenient to set a temporary root route, so it's easier to develop.

Devise also recommends you set one [when installing](https://github.com/heartcombo/devise#controller-filters-and-helpers), since users will redirect there after signing in/up.